Library/Homebrew/shims/super/cc

```diff
@@ -326,6 +328,9 @@ class Cmd
     end
     args += rpath_flags("#{wl}-rpath=", rpath_paths)
     args += ["#{wl}--dynamic-linker=#{dynamic_linker_path}"] if dynamic_linker_path
+    # Use -rpath-link to make sure linker uses glibc@2.13 rather than the system glibc for indirect
+    # dependencies because -L will only handle direct dependencies.
+    args << "#{wl}-rpath-link=#{@opt}/glibc@2.13/lib" if @deps.include?("glibc@2.13")
```

Comment:
Do we need to change the `-B` a few lines above?

Reply:
I guess it would only matter if the user was building against `glibc@2.13` on a host where brewed `glibc` was also installed.  Given that we're only building against `glibc@2.13` for relocatable bottles I don't think scenario should happen very often.  We could probably accommodate this by just doing:
```
if @deps.include?("glibc@2.13")
  args << "-B#{@opt}/glibc@2.13/lib"
else
  args << "-B#{@opt}/glibc/lib"
end
```
If this sounds good I'll push the change.

Reply:
LGTM

Reply:
Did you decide against pushing this?

Reply:
I suspect @danielnachun forgot.